libpci/src/pci.c

```diff
@@ -132,11 +132,6 @@ static void lib_pci_scan_bus(int bus);
 static void lib_pci_scan_fun(int bus, int dev, int fun)
 {
```

Comment:
Why not just get rid of this function, and call `libpci_add_fun()` directly from the scan?